alliance/config/settings/base.py

```diff
@@ -140,8 +160,9 @@ def template_folder_paths(app):
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': PROJECT_APP_TEMPLATES + BASE_TEMPLATES + EXTENSION_TEMPLATES,
-        'APP_DIRS': True,  # TODO
+        #'DIRS': PROJECT_APP_TEMPLATES + BASE_TEMPLATES + EXTENSION_TEMPLATES,
+        'DIRS': [os.path.join(PROJECT_ROOT, 'apps/shared/templates')],
```

Comment:
I agree this is less extensible. I was just going for the shortest path to working deployment so our teams can go ahead and use the app. I can't remember the errors I got before this change.